src/main/scala/tech/cryptonomic/conseil/application/LorreBakerRightsOperations.scala

```diff
@@ -0,0 +1,85 @@
+package tech.cryptonomic.conseil.application
+
+import tech.cryptonomic.conseil.util.IOUtils.{lift, IOLogging}
```

Comment:
`lift` is unused and few lines below `Timer`

Reply:
Thanks, I'll check it out